README.md

```
@@ -33,36 +33,36 @@ This approach aims to:
 
 
 ## Project Structure
 
 ### [Packages](./packages/README.md)
 
-- [**Runnable v0.1 package chooser**](./packages/README.md) - the generated install map for direct-install packages, support/runtime packages, and proof-host surfaces.
+- [**AppSurface v0.1 package chooser**](./packages/README.md) - the generated install map for direct-install packages, support/runtime packages, and proof-host surfaces.
 
-### [Core](./ForgeTrust.Runnable.Core/README.md)
-- [**ForgeTrust.Runnable.Core**](./ForgeTrust.Runnable.Core/README.md) – Core abstractions for defining modules and starting an application via `RunnableStartup` and `StartupContext`.
+### [Core](./ForgeTrust.AppSurface.Core/README.md)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)